src/Benchmarks/Configuration/WebHostBuilderExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+
+namespace Benchmarks.Configuration
+{
+    public static class WebHostBuilderExtensions
+    {
+        private static readonly string ServerKey = "server";
+        private static readonly string KestrelThreadCountKey = "threadCount";
+        private static readonly string Kestrel = "Kestrel";
+        private static readonly string WebListener = "WebListener";
+        public static IWebHostBuilder UseServer(this IWebHostBuilder builder, IConfigurationRoot config, out string serverName)
+        {
+            var serverValue = config[ServerKey] ?? Kestrel;
```

Comment:
I think it would be clearer to put this code directly in `Program.Main()`, rather than creating an extension method.